### DISCUSSION TITLE
How to add an extension to the wiki?

### DISCUSSION THREAD

**Discussion by ibhagwan** [mfussenegger/nvim-dap#441] [Q&A] [answered]
Trying to add [fzf-lua](https://github.com/ibhagwan/fzf-lua) to the extensions page after [adding support for `nvim-dap`](https://github.com/ibhagwan/fzf-lua/issues/101):

![image](https://user-images.githubusercontent.com/59988195/154128211-9cfc9eeb-2144-4fee-bd23-f659b2aaba74.png)

![image](https://user-images.githubusercontent.com/59988195/154128272-2f02e4df-a504-451a-8dac-52c5ad15adbb.png)

![image](https://user-images.githubusercontent.com/59988195/154128730-b7787006-d969-4517-ae79-dd92cdec1910.png)

I'm sure there's a straight forward answer for this...

The README says the wiki is community maintained but I couldn't find a way to add new items to the exensions page, does the same fork/PR route applies? I don't believe the wiki is part of the fork.

I tried git clone / push but this obviously fails as I have no permissions.

**Comment by mfussenegger** [comment, marked as answer]
You should see an edit button on each page - in the right upper corner:

![image](https://user-images.githubusercontent.com/38700/154144413-26abdecd-9b65-493b-8b8d-2cd4f92ae468.png)


Currently the editing is unrestricted, so any logged in user should be able to make edits.
